recipes/liquorice/pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
```

Comment:
Please remove this file, setup.cfg and setup.py and then move the meta.yaml into this directory.